pkg/collector/corechecks/oracle/sysmetrics_test.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 func TestSysmetrics(t *testing.T) {
+	t.Skip("sysmetrics can take several minutes to return the correct number")
```

Comment:
Could it be that the time is exactly two minutes (two snapshots 60s apart)? If the initialization time is deterministic, we could hard code it.

Reply:
I had some luck with reducing collection time and calling `Run` multiple times but it feels a bit fragile. Going to leave it for a later improvement.